android/build.gradle

```diff
@@ -8,9 +8,9 @@ buildscript {
         targetSdkVersion = 29
         supportLibVersion = "28.0.0"
         kotlinVersion = "1.3.61"
+        kotlin_version = "1.3.61"
```

Comment:
👀

Reply:
I think that's just a space.  But it is compiling and running fine on Android device.